geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -378,8 +378,11 @@ void doClear(RegionEventImpl regionEvent, boolean cacheWrite) {
     } finally {
       releaseDistributedClearLock(lockName);
       if (partitionedRegion.getCachePerfStats() != null) {
```

Comment:
I think that the null check on `partitionedRegion.getCachePerfStats()` is no longer necessary with this change, since it's not them we're updating.